СС-18569 Fix docs - Switch from Yarn to NPM (#1149)
spryker/spryker-docs: docs/scos/dev/front-end-development/migration-guide-switch-from-yarn-to-npm.md

{% info_block infoBox "Note" %}

To ensure the CI jobs run successfully, add the same config part to all `deploy.*.yml` files used by the frontend.

{% endinfo_block %}

2. The following commands are deprecated and work correctly only with the previous versions (using Yarn): `frontend:yves:install-dependencies`, `frontend:zed:install-dependencies`, `frontend:mp:install-dependencies`. Therefore, in all `*.yml` files at the project level, do the following:
   1. Remove the deprecated `frontend:yves:install-dependencies` and `frontend:zed:install-dependencies` commands.
   2. Replace the `frontend:mp:install-dependencies` command with `frontend:project:install-dependencies`.

3. Pass all (yves, zed, mp) frontend dependencies:

```bash
frontend:project:install-dependencies
```

4. Pull the latest `docker-sdk` version:

```bash
cd docker && git pull origin master
git log
```

> The `git log` command prints the table with the last three commits, copies the `hash` of the first commit, and replaces the content in the `.git.docker` file:

Author: This command only prints commits, need to copy paste hash manually

{% endinfo_block %}

> 2. The following commands are deprecated and will work correctly only with the previous versions (using Yarn). 

Reviewer: What do you mean by "the following commands"? Only commands from lines 46 and 47 below?
Remove deprecated `frontend:yves:install-dependencies` and `frontend:zed:install-dependencies` commands.
Replace the `frontend:mp:install-dependencies` command with the next one `frontend:project:install-dependencies`.

Author: These commands are deprecated 
```
frontend:yves:install-dependencies
frontend:zed:install-dependencies
frontend:mp:install-dependencies
```

Now we are using only one command to install all dependencies
```
frontend:project:install-dependencies
```

{% endinfo_block %}

> 2. The following commands are deprecated and will work correctly only with the previous versions (using Yarn). Please check and remove deprecated commands in all `*.yml` files.
> 
> ```bash
> frontend:yves:install-dependencies
> frontend:zed:install-dependencies
> ```
> 
> And replace `frontend:mp:install-dependencies` with the next one `frontend:project:install-dependencies`.

Reviewer 2: ```suggestion
2. The following commands are deprecated and will work correctly only with the previous versions (using Yarn). 
Please check all `*.yml` files at the project level and:
- Remove deprecated `frontend:yves:install-dependencies` and `frontend:zed:install-dependencies` commands.
- Replace the `frontend:mp:install-dependencies` command with the next one `frontend:project:install-dependencies`.
```
